.github/workflows/pr-check.yml

```diff
@@ -24,3 +24,5 @@ jobs:
         run: yarn run check
       - name: Run all tests
         run: yarn run test
+      - name: Build All packages
+        run: make build
```

Comment:
Wondering if we can split these up so changes in /api don't trigger a build for /web

Reply:
DONE